distage/distage-core-api/src/main/scala/izumi/distage/model/effect/DIEffectAsync.scala

```diff
@@ -93,7 +112,7 @@ private[effect] sealed trait LowPriorityDIEffectAsyncInstances {
     *
     * Optional instance via https://blog.7mind.io/no-more-orphans.html
     */
-  implicit final def fromParallelTimer[F[_], P[_[_]]: _Parallel, T[_[_]]: _Timer](implicit P: P[F], T: T[F]): DIEffectAsync[F] = {
+  implicit final def fromCats[F[_], P[_[_]]: _Parallel, T[_[_]]: _Timer, C[_[_]]: _Concurrent, M[_[_]]: _Functor](implicit P: P[F], T: T[F], C: C[F], M: M[F]): DIEffectAsync[F] = {
```

Comment:
Concurrent extends Functor, so additional Functor is redundant

Reply:
Yup, fixed it already.